source/_integrations/todo.markdown

```diff
@@ -0,0 +1,105 @@
+---
+title: To-do Lists
+description: Instructions on how to use To-do Lists within Home Assistant.
+ha_domain: todo
+ha_release: 2023.10
+ha_category:
+  - To-do List
+ha_quality_scale: internal
+ha_codeowners:
+  - '@home-assistant/core'
+ha_integration_type: entity
+---
+
+The To-do List integration provides todo list entities, allowing other integrations
+to integrate To-do Lists into Home Assistant. To-do lists are shown on the To-do list
+dashboard for tracking items and whether or not they have been completed.
+
+<div class='note'>
+
+To-do List entities are here to be consumed and provided by other integrations.
```

Comment:
We have a template now that we can use that explains platforms. See other integrations that allow platforms.